src/app/tests/suites/certification/Test_TC_TBRM_2_5.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name:
```

Comment:
Test cannot land until https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10264 is merged. This comment is here to place-hold for that merge.